### PR TITLE
API-13055 populate MVIPersonFullICN column for Practitioner.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**

### CODE REVIEW

File: minimart-manager/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <packaging>jar</packaging>
   <properties>
     <apache-csv.version>1.8</apache-csv.version>
-    <data-query.version>4.0.48</data-query.version>
+    <data-query.version>4.0.49-SNAPSHOT</data-query.version>
```
Comment (taylorrassmannlibertyits) [inline minimart-manager/pom.xml:15]: Snapshot version to be removed once [DQ PR](https://github.com/department-of-veterans-affairs/health-apis-data-query/pull/865) is merged

**Comment by taylorrassmannlibertyits** [comment]
FYI @nmoisan @jtaylor-va @haidong

File: minimart-manager/pom.xml (hunk 1 above)
Comment (jblefkove-va) [reply minimart-manager/pom.xml:15]: The DQ PR to update the entity is dependent on the new column being available in the database; this PR has to merge first. This PR should include only the DB changeset, with the pom update and MMM deferred to a separate PR

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**